### PR TITLE
Collections app

### PR DESCRIPTION
**Pull request by jamesgorrie**
Basic app for collections.
This remove the `/collections` endpoint form the metadata app, but so what.

### CODE REVIEW

File: collections/app/lib/Config.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
+
+  val appName = "collections"
+
+  val properties = Properties.fromPath("/etc/gu/collections.properties")
+
+  val awsCredentials: AWSCredentials =
+    new BasicAWSCredentials(properties("aws.id"), properties("aws.secret"))
+
+  val dynamoRegion: Region = Region.getRegion(Regions.EU_WEST_1)
```
Comment (akash1810) [inline collections/app/lib/Config.scala]: Just realised we're hard coding the region. Presumably if I was to create a stack in another region nothing would work?

File: collections/app/lib/Config.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  val keyStoreBucket = properties("auth.keystore.bucket")
+
+  val collectionsBucket: String = properties("s3.collections.bucket")
```
Comment (akash1810) [inline collections/app/lib/Config.scala]: does this bucket need creating in cf?

File: collections/app/lib/Config.scala (hunk 1 above)
Comment (theefer) [inline collections/app/lib/Config.scala]: I would not spend too much time worrying about that. That's an easy PR if someone is interested.

File: collections/conf/deploy.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "defaultStacks": ["media-service"],
+    "packages": {
+        "metadata-editor": {
```
Comment (theefer) [inline collections/conf/deploy.json]: That should be `collections` I think?

File: collections/app/lib/Config.scala [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  val imageCollectionsTable = properties("dynamo.table.imageCollections")
```
Comment (theefer) [inline collections/app/lib/Config.scala]: Will this need creating in CF?

File: project/Build.scala [hunk 5]
```diff
@@ -57,6 +57,10 @@ object Build extends Build {
     .libraryDependencies(awsDeps ++ imagingDeps)
     .testDependencies(scalaTestDeps)
 
+  val collections = playProject("collections")
+    .libraryDependencies(awsDeps ++ playWsDeps)
```
Comment (theefer) [inline project/Build.scala]: Do you actually use `WS` (I didn't check).

**Comment by theefer** [comment]
Nice one! :+1: 

Can I bother you with adding it to the `run*.sh` and `pingler` scripts in the root as well for completion?

File: collections/app/lib/Config.scala (hunk 2 above)
Comment (jamesgorrie) [inline collections/app/lib/Config.scala]: It does [locally](https://github.com/guardian/grid/blob/master/cloud-formation/dev-template.json#L219-L240) and [publicly](https://github.com/guardian/grid-infra/blob/master/platform/cloud-formation/media-service.json#L609-L630).

File: project/Build.scala (hunk 5 above)
Comment (jamesgorrie) [inline project/Build.scala]: Cheers, another copy paste

File: collections/conf/deploy.json (hunk 3 above)
Comment (jamesgorrie) [inline collections/conf/deploy.json]: Copy paste :rage4:

File: collections/app/lib/Config.scala (hunk 4 above)
Comment (jamesgorrie) [inline collections/app/lib/Config.scala]: [We do locally](https://github.com/guardian/grid/pull/1499/files#diff-a9f99695fda2ccf6f8fd009098777753R278) (which is what we need first) - I am adding the whole stack to the CF now.

**Comment by jamesgorrie** [comment]
@theefer added - I still run crappy OSX so don't really use those so forgot.

File: collections/app/lib/Config.scala (hunk 2 above)
Comment (akash1810) [inline collections/app/lib/Config.scala]: ah! I didn't check master :blush:

**Comment by jamesgorrie** [comment]
@kenoir I am using 9010, which is greater than than 9009, I just typoed pingler.

**Pull request merged**